lua/luasnip/init.lua

```diff
@@ -302,19 +303,15 @@ local function expand(opts)
 		local jump_into_func = opts and opts.jump_into_func
 
 		local cursor = util.get_cursor_0ind()
+
 		-- override snip with expanded copy.
 		snip = snip_expand(snip, {
 			expand_params = expand_params,
 			-- clear trigger-text.
-			clear_region = {
-				from = {
-					cursor[1],
-					cursor[2] - #expand_params.trigger,
-				},
-				to = cursor,
-			},
+			clear_region = expand_params.trig_region,
```

Comment:
I think having the length of the trigger as a default here was good, otherwise every `context_matcher` which does not modify the clear_region has to include this boilerplate piece of code

Reply:
make sense! fix it!